src/types.ts

```diff
@@ -590,7 +590,20 @@ export interface OBSEventTypes {
 		/**
 		 * The specific state of the output
 		 */
-		outputState: string;
+		outputState: 'OBS_WEBSOCKET_OUTPUT_STARTING' | 'OBS_WEBSOCKET_OUTPUT_STOPPING';
+	} | {
+		/**
+		 * Whether the output is active
+		 */
+		outputActive: boolean;
+		/**
+		 * The specific state of the output
+		 */
+		outputState: 'OBS_WEBSOCKET_OUTPUT_STARTED' | 'OBS_WEBSOCKET_OUTPUT_STOPPED';
```

Comment:
As mentioned at the start of this file, this file is auto generated and when there's a new release changes like this will not survive.

Either implement this to obs-websocket repo so [docs/generated/protocol.json](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.json) has this info that the generator can then read (preferred), or update [scripts/build-types.ts](https://github.com/obs-websocket-community-projects/obs-websocket-js/blob/master/scripts/build-types.ts) so the changes persist